Source/AliveLibAE/Renderer/stb_image.h

```diff
@@ -0,0 +1,7762 @@
+/* stb_image - v2.26 - public domain image loader - http://nothings.org/stb
```

Comment:
can we add this as a submodule? btw I think its alreading in ImGui so you can include it from there, 7k lines has nuked the diff view :D

Reply:
Dear Imgui doesn't include it unfortunately.

`Embeds stb_textedit.h, stb_truetype.h, stb_rect_pack.h by Sean Barrett (public domain).`

But yes submodule would be a good idea.